Flaskerizer_src/gui_for_flaskerizer.py

```diff
@@ -0,0 +1,185 @@
+        self.js_location.set(
+            "Select one JavaScript (.js) file from the template JavaScript folder")
+        self.main_layout()
+        self.is_test = is_test
+        if not self.is_test:        #For testing purposes, make it unable to open the window.
+            self.root.mainloop()
+        else:
+            pass
+
+    def main_layout(self):
+        """The GUI layout, defines all the buttons,
+        labels, and entries seen on the window.
+        """
+
+        self.main = tk.Frame(self.root)
+        self.label_html = tk.Label(self.main,
+                                   text="HTML File Location:")
+        self.label_html.grid(row=0, column=0, sticky="WENS", pady=20)
+        self.folder_search_html = tk.Button(self.main,
+                                            text="Search Files", command=self.get_html_folder)
+        self.folder_search_html.grid(row=0, column=1, sticky="WENS", pady=20)
+        self.folder_location_html = tk.Entry(self.main,
+                                             textvariable=self.html_location, width=80)
+        self.folder_location_html.grid(row=0, column=2, sticky="WENS", pady=20)
+
+
+        self.label_static = tk.Label(self.main,
+                                     text="Static Folder Location:")
+        self.label_static.grid(row=1, column=0, sticky="WENS", pady=20)
+        self.folder_search_static = tk.Button(self.main,
+                                              text="Search Files", command=self.get_static_folder)
+        self.folder_search_static.grid(row=1, column=1, sticky="WENS", pady=20)
+        self.folder_location_static = tk.Entry(self.main,
+                                               textvariable=self.static_location, width=80)
+        self.folder_location_static.grid(row=1, column=2, sticky="WENS", pady=20)
+
+
+        self.label_js = tk.Label(self.main,
+                                 text="JavaScript File Location:")
+        self.label_js.grid(row=2, column=0, sticky="WENS", pady=20)
+        self.folder_search_js = tk.Button(self.main,
+                                          text="Search Files", command=self.get_js_folder)
+        self.folder_search_js.grid(row=2, column=1, sticky="WENS", pady=20)
+        self.folder_location_js = tk.Entry(self.main,
+                                           textvariable=self.js_location, width=80)
+        self.folder_location_js.grid(row=2, column=2, sticky="WENS", pady=20)
+
+
+        self.main.pack()
+        self.ok_button = tk.Button(self.root,
+                                   text="OK", command=self.get_values).pack()
+
+    def get_html_folder(self):
+        """Opens a file dialog prompting the user to select an HTML file.
+        Gets the directory path with the path_to_folder function
+        and sets the html_location variable with the returned path value.
+        """
+
+        path = filedialog.askopenfilename(title= "Select the main html file normally index.html",
```

Comment:
Should the title be consistent with "Select one HTML file from the template" because we changed the prompt? I am actually not sure.  Same for the others.

Reply:
Yes it is the title of the window i think it should be consistent.
It could even have more information since it has more space and etc.